backend/src/impl/db_utils/system_db_utils.py

```diff
@@ -104,6 +104,23 @@ def system_from_dict(
         if "metric_stats" in document:
             metric_stats = document["metric_stats"]
             document["metric_stats"] = []
+
+        # FIX (lh): The following for loop is added to work around an issue related
```

Comment:
I think `TODO(user):` or `FIXME(user):` (TODO or FIXME followed by a username without whiltespaces) is better for searchability.